io.brackets.Brackets.json

```diff
@@ -0,0 +1,66 @@
+{
+  "app-id": "io.brackets.Brackets",
+  "branch": "stable",
+  "base": "io.atom.electron.BaseApp",
+  "base-version": "stable",
+  "runtime": "org.freedesktop.Sdk",
+  "runtime-version": "1.6",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "brackets",
+  "rename-desktop-file": "brackets.desktop",
+  "rename-icon": "brackets",
+  "separate-locales": false,
+  "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=pulseaudio",
+      "--share=network",
+      "--device=dri",
+      "--filesystem=host",
+      "--talk-name=org.freedesktop.Notifications",
+      "--talk-name=org.freedesktop.secrets",
+      "--talk-name=org.gtk.vfs",
+      "--talk-name=org.gtk.vfs.*",
+      "--talk-name=org.gnome.GConf",
+      "--filesystem=host"
+  ],
+  
+  "modules": [
+    {
+      "name": "brackets",
+      "buildsystem": "simple",
+      "build-commands": [
+        "ar x Brackets.Release.release-1.13-prerelease-3.64-bit.deb",
+        "tar -xf data.tar.xz",
+        "cp -r opt/brackets/* /app",
+
+        "mkdir -p /app/share/icons/hicolor/512x512/apps",
+        "mv usr/share/icons/hicolor/scalable/apps/brackets.svg /app/share/icons/hicolor/512x512/apps",
+	"rm -rf /app/share/icons/HighContrast",
+	
+	"ln -sf /app/brackets /app/bin/brackets",
+	"sed -i 's|\"$HERE/Brackets\"|\"$HERE/Brackets\" --no-sandbox|g' /app/brackets",
+	
+        "desktop-file-edit --set-key=Exec --set-value='brackets %F' --set-key=Icon --set-value=/app/share/icons/hicolor/512x512/apps/io.brackets.Brackets.svg /app/brackets.desktop",
+	"mkdir -p /app/share/applications/",
+	"install -Dm644 /app/brackets.desktop /app/share/applications/brackets.desktop",
```

Comment:
Install it under correct name instead of using `rename-*` (also can be moved instead of copying, but that's up to you).